scholia/app/views.py

```diff
@@ -1857,3 +1856,8 @@ def show_about():
 
     """
     return render_template('about.html')
+
+
+@main.route('/favicon.ico')
+def show_test():
```

Comment:
I am not sure why this is called show_test and not, e.g., show_favicon.

Reply:
good catch I called it show test as I was testing how flask routing works, I'll change it